### ISSUE TITLE
Door/window sensors not visible 

### ISSUE THREAD

**Issue by EricMuhren** [rvdvoorde/domoticz-homewizard#28]
Hi,

I'm new with Domoticz. I installed the plugin and all switches, energy, blinds and weather gadgets are working. But... I'm not able to see the door/window sensors in Domoticz. Can someone explain how to fix this? I can't find any related articles on the internet.

**Comment by c4coer** [comment]
quel le modèle des capteurs, car chez moi aucun probleme de ce type

**Comment by EricMuhren** [comment]
Hi,

I use KlikaanKlikuit ans Smartwares sensors. I think I found the problem in the plugin.py program. My sensor_type and sensor_name are returned as "unknown". There's is no "else" part in the program to handle these kind of sensors (unknown,unknown). So I changed the program a little bit and now it seems to work.

**Comment by c4coer** [comment]
tu peux partager, quel partie as tu changer merci

**Comment by c4coer** [comment]
quel partie as tu changé

**Comment by EricMuhren** [comment]
Hi,

I changed the code in this particular place. It's not generic, but it works for me. I think about making it generic.... It's possible to change the type of the sensor in Domoticz. So a door sensor I changed in "door contact" and a smoke switch I changed in "smoke detector". Now everything is working fine...!

Code added:
![code](https://user-images.githubusercontent.com/43743414/52919194-0c3f4380-3300-11e9-80a8-c9fa76929c26.JPG)
